src/components/run-button-container.js

```diff
@@ -261,6 +272,16 @@ export function RunButtonContainer({
                         headerId: 'startShortCircuitError',
                     });
                 });
+        } else if (action === runnable.VOLTAGE_INIT) {
+            setVoltageInitStatusState(RunningStatus.RUNNING);
+            startVoltageInit(studyUuid, currentNode?.id).catch((error) => {
+                console.info('failed');
```

Comment:
Maybe make the message a bit more explicit

Reply:
I removed the console.info